packages/design-preview/src/components/sidebar.tsx

```diff
@@ -93,23 +90,7 @@ const Sidebar: React.FC< SidebarProps > = ( {
 						) }
 						{ ( decodedDescription || decodedShortDescription ) && (
 							<div className="design-preview__sidebar-description">
-								<p>
-									{ isShowDescriptionToggle ? (
-										<>
-											{ isShowFullDescription ? decodedDescription : decodedShortDescription }
-											<Button
-												borderless
-												onClick={ () => setIsShowFullDescription( ! isShowFullDescription ) }
-											>
-												{ isShowFullDescription
-													? translate( 'Read less' )
-													: translate( 'Read more' ) }
```

Comment:
I guess we can keep this behavior for the theme only has one screen. WDYT?

Reply:
Hmm, but the reason we have the underlying issue is that we don't want to show empty sidebar :')

But that's interesting, this "Read less/more" behavior is no longer in production in the first place (even before this PR), do you know why we removed it?

Reply:
I'm not sure. @taipeicoder might have more context 🤔

Reply:
We never "supported" read less/more for theme previews in the onboarding design picker (see first [commit](https://github.com/Automattic/wp-calypso/commit/8a6611c48f5fbce0440131385f6fee6f51ea59d7) that introduces that component).

The read less/more was something we implemented for the theme preview in the in Theme Detail page, which we no longer use (since most information in the sidebar felt redundant) and kept the old theme preview instead.

Reply:
To provide even more context - there was a lot of discussion regarding showing the full theme description or just truncating it to the first sentence. In the end we decided to offer the read less/more toggle, but some theme description were reaaaally long which is why we (or rather I) didn't end up adding it to the design picker. Just didn't seem like an important thing to add.

Reply:
Got it. Thanks for the extra context 🙏 I guess we don't need it in this case either 😄